src/reducers/userReducer.js

```diff
@@ -3,7 +3,7 @@ import ActionTypes from '../actions/actionTypes';
 export function userReducer(state = {
     error: null,
     success: false,
-    user: null
+    user: {}
```

Comment:
Setting the user to null caused some unintended errors throughout the app, i feel like it should just be an empty object.